ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__automated-release-stable-4.16-upgrade-from-stable-4.16.yaml

```diff
@@ -49,6 +49,14 @@ releases:
       version_bounds:
         lower: 4.16.0-0
         upper: 4.17.0-0
+  source:
```

Comment:
stable -> stable does not run any test suite, so we do not need the setting here.

Reply:
remove it, thanks